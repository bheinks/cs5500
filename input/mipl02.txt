(*
	This file has a lexical error.
*)

program DiddleDeDum;
begin (* DiddleDeDum *)
  x := 1234;
  write('d', 'o', 'g,'\')
end (*DiddleDeDum *) .
