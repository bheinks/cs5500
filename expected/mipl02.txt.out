TOKEN: T_IDENT     LEXEME: progrm
Line 5: syntax error
